eng/common/pipelines/templates/steps/detect-api-changes.yml

```diff
@@ -0,0 +1,31 @@
+parameters:
+  ArtifactPath: $(Build.ArtifactStagingDirectory)
+  Artifacts: []
+
+steps:
+  - pwsh: |
+      $artifacts = '${{ convertToJson(parameters.Artifacts) }}' | ConvertFrom-Json
+      $artifactList = @()
+      foreach ($artifact in $artifacts) {
+        $artifactList += "$($artifact.name)"
+      }
+      $artifactListValue = $artifactList -join ','
+
+      Write-Host "artifactListValue = $artifactListValue"
+      Write-Host "##vso[task.setvariable variable=artifactList;]$artifactListValue"
+    displayName: Initialize artifacts list variable
+
+  - task: Powershell@2
+    inputs:
+      filePath: $(Build.SourcesDirectory)/eng/common/scripts/Detect-Api-Changes.ps1
+      arguments: >
+        -ArtifactList '$(artifactList)'
+        -ArtifactPath ${{parameters.ArtifactPath}}
+        -CommitSha '$(Build.SourceVersion)'
+        -BuildId $(Build.BuildId)
+        -PullRequestNumber $(System.PullRequest.PullRequestNumber)
```

Comment:
Don't you want to pass in FullRepoName setting it to ` $(Build.Repository.Name)`?